Sources/ApodiniNetworking/ChannelHandlers/RequestDecoder.swift

```diff
@@ -51,7 +51,16 @@ class HTTPServerRequestDecoder: ChannelInboundHandler, RemovableChannelHandler {
             if req.headers[.contentLength] == bodyBuffer.readableBytes {
                 req.bodyStorage = .buffer(bodyBuffer)
                 state = .awaitingEnd(req)
+            } else if req.headers[.transferEncoding].contains(.chunked) {
+                // The request body is chunked, so we need to take care of that.
+                fatalError("Not yet implemented: Incoming requests with 'Transfer-Encoding: chunked' not yet supported")
```

Comment:
Yeah good point, I'll change it to a 500 error response.

Thing is, supporting chunked requests isn't actually required anymore, since the HTTP interface exporter's streaming is implemented on top of HTTP/2, and Transfer-Encoding is an HTTP/1-only thing.